### PR TITLE
Add recv_buffer parameter for in_udp

### PR DESCRIPTION
**Pull request by mururu**
It is useful for tuning of udp input.

### CODE REVIEW

File: lib/fluent/plugin/in_udp.rb [hunk 1]
```diff
@@ -40,6 +40,8 @@ class UdpInput < Input
     config_param :message_length_limit, :size, default: 4096
     desc "Remove newline from the end of incoming payload"
     config_param :remove_newline, :bool, default: true
+    desc "The max size of socket receive buffer. SO_RCVBUF"
+    config_param :recv_buffer, :size, default: nil
```
Comment (repeatedly) [inline lib/fluent/plugin/in_udp.rb]: `receive_buffer_size` is better.

Comment (mururu) [reply lib/fluent/plugin/in_udp.rb]: Okay. I just followed `recv_timeout`, but it isn't a parameter name yet.

**Comment by mururu** [comment]
The test fails on CI. I'll check it.

File: lib/fluent/plugin/in_udp.rb (hunk 1 above)
Comment (repeatedly) [reply lib/fluent/plugin/in_udp.rb]: Yeah, public name should be clear.

**Comment by repeatedly** [comment]
From man page, returned value is different from get/set: http://man7.org/linux/man-pages/man7/socket.7.html

**Pull request closed without merging**